### PR TITLE
WJ-176: Replace gender with pronouns

### PR DESCRIPTION
**Pull request by danieltharp**
This PR is primarily around replacing the `gender` field in the database (`char` type) with a `pronouns` field (`varchar(30)` type) and updating references to it in the templates and DB seed. It also includes a regression and feature test.

It also fixes some configuration around phpunit and updates a gitignore file to make it...work.

### CODE REVIEW

File: install/local/dev/postgres/seed.sql [hunk 1]
```diff
@@ -2850,20 +2850,15 @@ ALTER SEQUENCE public.private_user_block_block_id_seq OWNED BY public.private_us
...
     birthday_month integer,
     birthday_year integer,
     about text,
     location character varying(70),
     website character varying(100),
-    im_aim character varying(100),
-    im_gadu_gadu character varying(100),
-    im_google_talk character varying(100),
     im_icq character varying(100),
     im_jabber character varying(100),
```
Comment (emmiegit) [inline install/local/dev/postgres/seed.sql:2861]: Honestly, I would cut all of them. I doubt the number of people active on Wikidot who use either of these services is more than like, 3 people.

Comment (danieltharp) [reply install/local/dev/postgres/seed.sql:2861]: All I wanted to do with this one was prove out that stuff can be removed in the seed and Ozone ORM and have it work as expected, given known useless fields. We had a larger conversation about replacing this with a morphable `profile_fields` table that I would roll further changes into.

Comment (emmiegit) [reply install/local/dev/postgres/seed.sql:2861]: Ah interesting, okay. And yeah I agree long term that's what we'll be doing for profiles.

**Comment by danieltharp** [comment]
Marking this as ready as my actual browser testing works as expected with getting and setting these values.

**Review by emmiegit — APPROVED** [review]

**Pull request merged**